pom.xml

```diff
@@ -3,7 +3,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.github.pjfanning</groupId>
   <artifactId>excel-streaming-reader</artifactId>
-  <version>2.3.6-SNAPSHOT</version>
+  <version>2.3.7-SNAPSHOT</version>
```

Comment:
can you remove this? - I'll worry about the version numbers.

Reply:
Done